hera_mc/__init__.py

```diff
@@ -84,7 +84,9 @@ def NotNull(kind, **kwargs):
     return Column(kind, nullable=False, **kwargs)
 
 
-from .version import __version__  # noqa
+from . import version  # noqa
+__version__ = version.version
```

Comment:
Is there a reason why we have to define this here, and we can't wait until all the imports are done?

Reply:
This is the only place the `__version__` attribute is defined. Oh! you mean move that line after the imports. Sure, I can do that.